Tests/TestUser.php

```diff
@@ -43,7 +43,7 @@ public function getPassword()
      */
     public function getSalt()
     {
-        return null;
```

Comment:
Return `null` is correct

> This can return null if the password was not encoded using a salt.
http://api.symfony.com/2.4/Symfony/Component/Security/Core/User/AdvancedUserInterface.html#method_getSalt